docs/Using-Fleet/Permissions.md

```diff
@@ -94,6 +94,7 @@ GitOps is an API-only and write-only role that can be used on CI/CD pipelines.
 
 \*** Applies only to [Fleet REST API](https://fleetdm.com/docs/using-fleet/rest-api)
 
+\**** Global admins can neither demote their own role nor change themselves to be a Team-specific user.
```

Comment:
Would leave "Team" lowercased here, since it's lowercased in the table above. (Not 100% sure what our conventions are for capitalizing names of features in Fleet; I've seen a mix of capitalized/uncapitalized.)

Reply:
@rachaelshaw will do